datafusion/expr/src/udaf.rs

```diff
@@ -635,6 +635,14 @@ pub trait AggregateUDFImpl: Debug + Send + Sync {
     fn documentation(&self) -> Option<&Documentation> {
         None
     }
+
+    /// Indicates whether the aggregation function is monotonic as a set function. A set
+    /// function is monotonically increasing if its value increases as its argument grows
+    /// (as a set). Formally, `f` is a monotonically increasing set function if `f(S) >= f(T)`
+    /// whenever `S` is a superset of `T`.
```

Comment:
I think it would help this documentation if it also gave some examples

```suggestion
    /// whenever `S` is a superset of `T`.
    ///
    /// For example `count` and `max` are monotonically ascending as their values always increase
    /// (or stay the same) as new values are seen. 
    ///
    /// `min` is monotonically descending as its value always decreases or
    /// stays the same as new values are seen. 
```